dags/imagegen_devx/jax_stable_stack_e2e.py

```diff
@@ -75,7 +75,7 @@
           test_name=f"maxtext-jax-stable-stack-{accelerator}-{slice_num}x",
           docker_image=DockerImage.MAXTEXT_TPU_JAX_STABLE_STACK.value,
           test_owner=test_owner.PARAM_B,
-      ).run()
+      ).run_with_quarantine()
```

Comment:
@parambole has split this DAG in https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/464/files. Maybe revert this file to avoid merge conflict?

Reply:
Hey @shralex https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/464/files is not merged. Ideally the merger should be smart enough to merge it given the split involves renaming this file. Let me know if you face any merge issues when you merge main to your current branch.

Reply:
Already reverted this file.